### ISSUE TITLE
JQuery UI Error on /bitcoinkeys

### ISSUE THREAD

**Issue by tadityar** [blockchain-certificates/cert-viewer#61]
Machine specs: Google Chrome on RHEL 7

Clicking "Generate Keys" on the `/bitcoinkeys` page does not produce the printable keys, after inspecting the page it says:
```
Uncaught TypeError: $(...).tooltip is not a function
    at HTMLDocument.<anonymous> (bitcoinkeys.js:4)
```

Googling reveals this is because the `.tooltip` function was not included. Updating the cdn url in `layout.html` to the latest jquery release solved the problem. I'm not sure if this is reproducible on other setups though.

**Comment by kimdhamilton** [comment]
Thanks Tasya! I've reprod the error, and I'll add this fix

**Comment by kimdhamilton** [comment]
Fixed in #62
